extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.example.1.json

```diff
@@ -6,17 +6,14 @@
           "xdm:id": "experience1"
         }
       ],
-      "xdm:decisionEventID": "TNT:activity1:experience1:1597208433",
-      "xdm:decisionScope": "__view__"
+      "xdm:id": "AT:eyJhY3Rpdml0eUlkIjoiMzg0NDU5IiwiZXhwZXJpZW5jZUlkIjoiMSJ9"
```

Comment:
Could we maybe remove this example, since it doesn't contain most of the data we'd need, and may be confusing?

Reply:
I removed it.